packages/shared/components/drawerContent/AccountSwitcher.svelte

```diff
@@ -52,7 +51,8 @@
                 class="hover:bg-gray-50 dark:hover:bg-gray-800 flex flex-row items-center space-x-4 p-4 rounded"
             >
                 <div class="circle" style="--account-color: {getColor($activeProfile, account.id)};" />
-                <Text secondary={account.id !== $selectedAccount?.id} type="{$mobile ? 'h5' : 'h4'}">{account.alias}</Text>
+                <Text secondary={account.id !== $selectedAccount?.id} type={$mobile ? 'h5' : 'h4'}>{account.alias}</Text
+                >
```

Comment:
Did this happen automatically? I prefer having the whole `</Text>` on the next line, so you can see faster what component ends.

Reply:
Maybe was because I added code from previous to the refactor I think 🤔 
Yes I also prefer as you said, will apply that.